README.md

```diff
@@ -29,9 +29,19 @@ Look through the application code. If you have the old Lambda Times (Applied Jav
 ## Self-Study/Essay Questions
 
 - [ ] What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+Used to make sure the correct type of data is being passed around. They are important because you or any other programmer can refer to them to see whats being passed around so that there are no mistakes. 
+
 - [ ] Describe a life-cycle event in React?
+A sequence of events that happen in a components life cycle. 
+
```

Comment:
These are methods can be called during the lifecycle of a component, and they allow you to update the UI and application states.

See _[this](https://medium.com/@joshuablankenshipnola/react-component-lifecycle-events-cb77e670a093)_ article.